### ISSUE TITLE
DBGate does not permit opening a Sqlite3 .db file located in a non-standard Linux directory.

### ISSUE THREAD

**Issue by CarlBean-USDA** [dbgate/dbgate#529]
I want to open a .db file (Sqlite3)  located off of a **/data** directory on my Ubuntu guest VM and DBGate does not allow this.  It looks like it only allows files to be opened off of the $HOME and the standard Linux dirs (that shows when I click on the computer icon on the Select dialog).  I thought this might be a permissions issue so I changed the ownership of the entire directory path to my $HOME account and it still would not allow this.

**Comment by ProjectInfinity** [comment]
Could you explain your setup a little more?

Are you running DbGate inside a VM?
Is /data a directory owned by the VM or is it shared from the host?
Which filepicker are you attempting to open with? (standard is GTK file picker, even on Qt desktops)

What happens if you symlink $HOME/data to /data, does it allow opening then?

**Comment by CarlBean-USDA** [comment]
Yes.  I'm running DbGate in a guest Ubuntu 20.04 VM (VMWare Workstation 16 Player) on a Windows 11 host.
The /data directory is owned by the VM (not shared from the host).  Specifically, all directories in the file path are owned by my $HOME account.
The file picker is the one that launches when running DbGate from Ubuntu 20.04.  I believe the default file picker on Ubuntu 20.04 is Gnome  Gtk.FileChooser.  Apparently, it does not show symbolic links.  I tried typing it in manually and it still would not recognize it.  I will look into seeing about replacing this file picker with one that does show symbolic links.
I created a symbolic link but could only see it when either doing a **ls** command (from the **Terminal** app) or using the **Files** app (which is Ubuntu's Windows Explorer equivalent).
